apps/admin/src/app/forms/forms-page.component.html

```
@@ -168,6 +168,65 @@ <h4>Identidade e sigilo</h4>
+              @if (workspace.uploadingImageTarget() === 'form') {
+                <mat-progress-bar mode="indeterminate" aria-label="Enviando imagem do formulário" />
+              }
+
+              @for (image of workspace.descriptionImages(); track $index) {
+                <div class="form-image-row">
+                  <lib-event-form-description-content [images]="[image]" />
+                  <div class="form-image-fields">
+                    <mat-form-field>
+                      <mat-label>Texto alternativo</mat-label>
+                      <input
+                        matInput
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)